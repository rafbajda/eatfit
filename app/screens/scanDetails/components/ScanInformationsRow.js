import React from 'react';
import { Row, Col } from 'native-base';
import ScanImage from './ScanImage';
import ScanInformations from './ScanInformations';

const ScanInformationsRow = props => {
    const { scanUrl, scanScore } = props;

    return (
        <Row style={{ height: 200 }}>
            <Col>
                {scanUrl ? <ScanImage scanUrl={scanUrl} /> : null}
            </Col>
            <Col>
                {scanScore ? <ScanInformations score={scanScore} /> : null}
            </Col>
        </Row>
    );
};

export default ScanInformationsRow;
