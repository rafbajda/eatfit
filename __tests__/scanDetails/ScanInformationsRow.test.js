import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import NavigationTestUtils from 'react-navigation/NavigationTestUtils';
import ScanInformationsRow from '../../app/screens/scanDetails/components/ScanInformationsRow';

describe('ScanInformationsRow snapshot', () => {
    jest.useFakeTimers();
    beforeEach(() => {
        NavigationTestUtils.resetInternalState();
    });

    it('renders component', async () => {
        const tree = renderer.create(<ScanInformationsRow />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
