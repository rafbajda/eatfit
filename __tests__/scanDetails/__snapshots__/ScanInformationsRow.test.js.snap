// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScanInformationsRow snapshot renders component 1`] = `
<View
  style={
    Object {
      "flex": 0,
      "flexDirection": "row",
      "height": 200,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "column",
      }
    }
  />
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "column",
      }
    }
  />
</View>
`;
